docs/guide/protocols/absolutesolvation.rst

```diff
@@ -1,4 +1,94 @@
+   :scale: 100%
+
+   Thermodynamic cycle for the absolute hydration free energy protocol.
+
+Scientific Details
+------------------
+
+Partial annhilation scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the :class:`.AbsoluteSolvationProtocol` the coulombic interactions of the molecule are fully turned off (annihilated),
+while the Lennard-Jones interactions are decoupled, meaning the intermolecular interactions turned off, keeping the intramolecular Lennard-Jones interactions.
+
+The lambda schedule
+~~~~~~~~~~~~~~~~~~~
+
+Molecular interactions are turned off during an alchemical path using a discrete set of lambda windows. The electrostatic interactions are turned off first, followed by the decoupling of the van-der-Waals interactions. A soft-core potential is applied to the Lennard-Jones potential to avoid instablilites in intermediate lambda windows. 
+
+.. _Simulation details:
+
+Simulation details
+~~~~~~~~~~~~~~~~~~
+
+The protocol applies a LangevinMiddleIntegrator which uses Langevin dynamics, with the LFMiddle discretization [1]_.
+Before running the production MD simulation in the NPT ensemble, the protocol performs a minimization of the system, followed by an equilibration in the NPT ensemble. A MonteCarloBarostat is used in the NPT ensemble to maintain constant pressure.
+
+Getting the free energy estimate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The free energy differences are obtained from simulation data using the MBAR estimator (multistate Bennett acceptance ratio estimator).
+TODO: Link to results page once done
+
+In addition to the MBAR estimates of the two legs of the thermodynamic cycle and the overall absolute solvation free energy difference, the protocol also returns some metrics to help assess convergence of the results. The forward and reverse analysis looks at the time convergence of the free energy estimates. The MABR overlap matrix checks how well lambda states overlap. Since the accuracy of the MBAR estimator depends on sufficient overlap between lambda states, this is a very important metric. 
+To assess the mixing of lambda states in the Hamiltonian replica exchange method, the results object returns the replica exchange transition matrix, which can be plotted as the replica exchange overlap matrix, as well as a time series of all replica states. (Todo: link to the results page in case examples of these plots are deposited there) 
+
+Simulation overview
+-------------------
+
+The ``ProtocolDAG`` of the :class:`.AbsoluteSolvationProtocol` contains both the units from the vacuum and from the solvent transformations. 
+This means that both legs of the thermodynamic cycle are constructured and run concurrently in the same ``ProtocolDAG``. This is different from the :class:`RelativeHybridTopologyProtocol` where the ``ProtocolDAG`` only runs a single leg of a thermodynamic cycle.
+If multiple repeats of the protocol are run, the ``ProtocolDAG`` contains multiple units of both vacuum and solvent transformations. 
```

Comment:
Here it might be better to specifically refer to `protocol_repeats` (and say that by default it is set to 3).

That way folks know it's more about "internal repeats" than "external repeats" (sorry that's a poor explanation).